HHS/Head-Start-TTADP
# Prod - Fix incorrect participant count for all regions (#377) and dashboard visual fixes (#375)

## Description of change
- The participant count as well as the total tta hours did not show the correct number when the "All regions" option was selected for the TTA Overview widget on the activity reports page. This PR fixes it.
- the topic/frequency graph had too much whitespace between the filters and the graph. The container wasn't staying static.
- Rename the link to the dashboard in the side navigation
- added an error message to the date range picker if a date isn't entered in the correct format, rather than silently correcting the date to the first available in the date range


## How to test
- navigate to https://tta-smarthub-staging.app.cloud.gov/ 
- verify that the "All regions" option shows the correct value (previously it would show the number for the first region only)
- Navigate to the dashboard and verify the changes above have been made and function appropriately.
## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-258
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-260


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
